charts/bindplane/templates/bindplane-jobs.yaml

```diff
@@ -59,8 +59,8 @@ spec:
       topologySpreadConstraints:
       {{- toYaml .Values.topologySpreadConstraints.jobs | nindent 8 }}
       {{- end }}
-      {{- if .Values.backend.postgres.sslsecret.name }}
```

Comment:
This condition is now nested below `initContainers`, to avoid removing the `initContainers` line when ssl secret name is not set.